okrs/2019-q3.md

```diff
@@ -0,0 +1,19 @@
+## Objective: Projects that depend on IPLD are using new interfaces and specifications
+
+
+| Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
+| ---------- | -------- | -------- | ----------- | --------------- | ----------- |
+| [API stable UnixFSv2 on JS Composites](https://github.com/ipld/js-unixfsv2/pull/3) | @mikeal | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| Complete selector work necessary for GraphSync | @hannahhoward | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| Schema based codegen works in go-ipld-prime | @warpfork | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+
+## Objective: Realize the vision of decentralized linkable advanced data structures
+
+| Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
+| ---------- | -------- | -------- | ----------- | --------------- | ----------- |
+| [JS Composite engine supports writes](https://github.com/ipld/js-composites/pull/4) | @mikeal | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| Resolve "fat pointers" for Composites open problem | @mikeal | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | | | |
+| Rust has a `dag-cbor` codec | @vmx | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| Rust can compile functions to WASM that can speak the complete data model to JS | @vmx | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | | | |
```

Comment:
I've troubles understanding that one. Is this about glue code between JS and WASM?

Reply:
Yes. More specifically, since there’s already a binding layer that supports all the types except for `Link`, this is mostly about having one of these existing JS to WASM bindings working and figuring out how we handle `Link` types.